index.html

```diff
@@ -5,8 +5,142 @@
+            <a class="navigation-link" href="#">О Седоне</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="catalog.html">Гостиницы</a>
+          </li>
+        </ul>
+        <ul class="navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Поиск</span>
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Избранное</span>
+            </a>
+          </li>
+          <li class="navigation-item">
+            <button class="button button-booking">Хочу сюда!</button>
+          </li>
+        </ul>
+      </nav>
+    </header>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+    <main class="main-index">
+      <h1 class="visually-hidden">Сайт города Седона</h1>
 
+      <section class="wellcome">
+        <h2 class="visually-hidden">Добро пожаловать в Седону!</h2>
+        <img src="" alt="Добро пожаловать в прекрасную Седону! Потому что Гранд-Каньон - отстой!">
+        <p><b>Седона — небольшой городок в Аризоне,
+          заслуживающий большего!</b></p>
+        <p>Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+      </section>
+
+      <section class="advantages">
+        <h2 class="visually-hidden">Приемущества:</h2>
+        <ol class="advantage-list">
+          <li class="advantage-item">
+            <span class="advantage-name">Настоящий городок</span><br>
+            <p class="advantage-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+            <img class="background" src="" alt="Городской пейзаж">
+            <h3 class="visually-hidden">Рекомендуем:</h3>
+              <ul class="advantage-list">
+                <li class="advantage-item">
+                  <span class="advantage-name">Жильё</span><br>
+                  <p class="advantage-description">Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+                </li>
+                <li class="advantage-item">
+                  <span class="advantage-name">Еда</span><br>
+                  <p class="advantage-description">Всегда заказывайте топовый фирменный бургер, вы не разочаруетесь!</p>
+                </li>
+                <li class="advantage-item">
+                  <span class="advantage-name">Сувениры</span><br>
+                  <p class="advantage-description">Не только китайского, но и настоящего местного производства!</p>
+                </li>
+              </ul>
+          <li class="advantage-item">
+            <span class="advantage-name">Там есть мост дьявола</span><br>
+            <p class="advantage-description"></p>Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
```

Comment:
Редактор похоже автозакрыл тэг
```suggestion
            <p class="advantage-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
```

Reply:
И на следующих пунктах тоже самое.